x-pack/test_serverless/functional/test_suites/common/reporting/management.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default ({ getPageObjects, getService }: FtrProviderContext) => {
+  // const esArchiver = getService('esArchiver');
+  // const log = getService('log');
+  // const pageObjects = getPageObjects(['common', 'security']);
+  // const retry = getService('retry');
+  // const testSubjects = getService('testSubjects');
+
+  describe('Reporting Management', function () {});
```

Comment:
There isn't much we can do in functional tests until we have the ability to log in as a non-`elastic` user.